memcache/memcache_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	"time"
 )
 
-const testServer = "localhost:11211"
+const testServer = "127.0.0.1:11211"
```

Comment:
NB: I had to use this to connect to my loopback-only memcached (we don't typically want local services to bind to external interfaces).  I can revert this line if you want, but thought it might actually be nice to include.